test/modes/search_test.exs

```diff
@@ -3,44 +3,84 @@ defmodule SonicClient.Modes.SearchTest do
   alias SonicClient.Modes.Search
   import SonicClient.TestConnectionHelper
 
+  setup_all do
+    add_data("user:1", "It is a common test")
+    add_data("user:2", "It is a common testable text")
+    add_data("user:3", "It should not appear in the common search")
+
+    on_exit(fn -> flush() end)
```

Comment:
Wait a minute, is `add_data` async, or something?

Reply:
Nah, I guess this is a test specific callback?

Reply:
`setup_all` is the test callback and `add_data` is function that is in the module `test/support/test_connection_helper.ex` which I'm importing in the test.

Reply:
@edgarlatorre I was referring to this `on_exit` function we pass an anonymous function to. I may be called at the end of the test suite?

Reply:
Ah sorry, yes it's a callback which in the end of the the tests it calls the function which was passed as argument.